addons/l10n_in/views/report_invoice.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="l10n_in_report_invoice_document_inherit" inherit_id="account.report_invoice_document">
+        <xpath expr="//div[@id='total']" position="after">
+            <div id="total_in_words" class="row mb16">
+                <div class="col-xs-2">
+                    <strong>Total (In Words): </strong>
+                </div>
+                <div class="col-xs-8">
+                    <span t-field="o.amount_total_words"/>
+                </div>
+            </div>
+        </xpath>
+        <xpath expr="//tbody[hasclass('invoice_tbody')]/tr/td[1]" position="replace">
+            <td><span t-field="l.name"/><t t-if="l.product_id.hsn_code"><h6><strong class="ml16">HSN/SAC Code:</strong> <span t-field="l.product_id.hsn_code"/></h6></t></td>
+        </xpath>
+        <xpath expr="//div[@name='invoice_address']//div[@t-if='o.partner_id.vat']" position="replace">
+            <div t-if="o.partner_id.vat" class="mt16">GSTIN: <span t-field="o.partner_id.vat"/></div>
```

Comment:
overwritting reports to replace the label VAT by GSTIN is not a good solution.

Please do the following:
* in base, on res.country, add a new field vat_label. not required, no default. translate=True
* set this field for european countries to 'VAT'
* set this field for India (and other countries using GST if you know them) to 'GSTIN'
* in reports (invoice, purchase, sale, picking...), display 
```<span t-field="o.company_id.country_id.vat_label"/>: <span t-field="o.partner_id.vat"/>```